bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Integration/SystemUsers/SystemUserClient.cs

```diff
@@ -75,7 +75,8 @@ private static SystemUserReal MapDescriptorToSystemUserReal(SystemUserDescriptor
         { 
             PartyId = newSystemUserDescriptor.OwnedByPartyId ?? string.Empty,
             IntegrationTitle = newSystemUserDescriptor.IntegrationTitle!,
-            ProductName = newSystemUserDescriptor.SelectedSystemType!            
+            SystemId = newSystemUserDescriptor.SelectedSystemType!,
```

Comment:
wondering if there are any similar changes needed in react app?

Reply:
systemId is collected from the selectedsystemtype, and the reporteeorgno is from the context handler